marimo/_plugins/ui/_impl/table.py

```diff
@@ -270,3 +281,6 @@ def get_column_summaries(self, args: EmptyArgs) -> ColumnSummaries:
             )
 
         return ColumnSummaries(summaries)
+
+    def sort_values(self, args: SortValuesArgs) -> List[JSONType]:
+        return self._manager.sort_values(args.by, args.descending).to_data()
```

Comment:
i think we will want to do:
```
self._unfiltered_manager.sort_values(args.by, args.descending).limit(TableManager.DEFAULT_LIMIT).to_data()
```

Reply:
Thanks, you're right. I didn't look into the init closely enough